cfn-resources/auditing/cmd/resource/resource.go

```diff
@@ -178,10 +186,9 @@ func Update(req handler.Request, prevModel *Model, currentModel *Model) (handler
 		}, nil
 	}
 
-	var res *mongodbatlas.Response
-
-	auditingInput := mongodbatlas.Auditing{}
-
+	var res *http.Response
+	enabled := true
+	auditingInput := admin.AuditLog{Enabled: &enabled}
```

Comment:
curious why previously update operation was not defining the enable property, should we preserve it that way?

Reply:
Updated. Yes, this was previously updated as enabled was required I believe, now that all are optional after changes in mms/SDK, have reverted to original implementation.